sw/device/silicon_creator/imm_rom_ext/BUILD

```diff
@@ -0,0 +1,11 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "main_lib",
```

Comment:
nit: if you named this `imm_rom_ext` then you could refer to it in build rules as `//sw/device/silicon_creator/imm_rom_ext`, instead of `//sw/device/silicon_creator/imm_rom_ext:main_lib`.

Reply:
(also OK with merging this as is; and it can be cleaned up later due to CI times at the moment). Just lmk if you would like me to merge it.

Reply:
Sure, let's clean it up in a later CL.